demo-simple/src/main/java/com/novoda/downloadmanager/demo/MainActivity.java

```diff
@@ -17,10 +18,9 @@
 import com.novoda.downloadmanager.LiteDownloadManagerCommands;
 import com.novoda.downloadmanager.LocalFilesDirectory;
 import com.novoda.downloadmanager.LocalFilesDirectoryFactory;
-import com.novoda.downloadmanager.MigrationFactory;
 import com.novoda.downloadmanager.MigrationServiceBinder;
```

Comment:
Can this be an interface instead? Also, what if we just rename these classes dropping the `binder` suffix?

Reply:
Suggested the same rename yesterday 😅

Reply:
Not even any real need for `Service` to be in there either. How about `DownloadMigrator`?